.travis.yml

```diff
@@ -1,3 +1,16 @@
-script:
-- docker build --target all_tests -t test . && docker run --name todo-app-test --env TRELLO_URL --env TRELLO_KEY --env TRELLO_BOARD_ID --env TRELLO_TOKEN test
-- docker build --target prod -t prod .
+jobs:
+  include:
+    - stage: test
+      script:
+        - docker build --target all_tests -t test .
+        - docker run --name todo-app-test --env TRELLO_URL --env TRELLO_KEY --env TRELLO_BOARD_ID --env TRELLO_TOKEN test
+    - stage: deploy
+      script:
```

Comment:
This stage currently runs on every branch (which makes sense for testing, but not once this is ready)

Note that just specifying this stage to run only on master won't be enough as by default the branch will be considered master for any PRs into master. You're probably best off using [the explicit deployment step](https://docs.travis-ci.com/user/deployment/script/) which has that default PR behaviour disabled